index.bs

```diff
@@ -4864,12 +4864,13 @@ Note: In case of an arrow function in <code>functionDeclaration</code>, the
       }
 
       script.CallFunctionParameters = {
-        functionDeclaration: text;
-        awaitPromise: bool;
-        target: script.Target;
-        ?arguments: [*script.ArgumentValue];
-        ?this: script.ArgumentValue;
-        ?resultOwnership: script.ResultOwnership;
+        functionDeclaration: text,
+        awaitPromise: bool,
+        target: script.Target,
+        ?arguments: [*script.ArgumentValue],
+        ?maxDepth: js-uint,
```

Comment:
I assume that as long it is not clear if we can use defaults here you wont further use them? Otherwise we could set a default of `1` already here and won't need the prose.